addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/service/MediaRendererService.java

```diff
@@ -182,7 +182,8 @@ public void onValueReceived(String variable, String value, String service) {
 
         String oldValue = stateMap.get(variable);
         if ((value == null && oldValue == null) || (value != null && value.equals(oldValue))) {
-            logger.trace("Value '{}' for {} hasn't changed, ignoring update", value, variable);
+            // called every second: a bit much to log each time
+            // logger.trace("Value '{}' for {} hasn't changed, ignoring update", value, variable);
```

Comment:
Should we then remove the commented out log statement as well?

Reply:
I am not certain here. It is useful for debugging while developing, but really bloats the logs. So I commented it out for myself and may still be useful for future debugging. What is the preferred practise?

Reply:
Decided to uncomment it. TRACE level logging is bloated quite a bit anyway, so this bit more is not really harmful.